src/bosh_aws_cpi/lib/cloud/aws/block_device_manager.rb

```diff
@@ -59,7 +59,7 @@ def self.requires_nvme_device(instance_type)
 
     def mappings(info)
       instance_type = @vm_cloud_props.instance_type.nil? ? 'unspecified' : @vm_cloud_props.instance_type
-      if instance_type =~ /^i3\./
+      if instance_type =~ /^i3\./ || instance_type =~ /^i3en\./
```

Comment:
This could also be written like: `/^i3(|en)\./`
https://rubular.com/r/8TyZH5ChBZlw6t

Reply:
I'm awful at Regex sometimes, thanks.